src/core/types/addons.js

```diff
@@ -1,4 +1,9 @@
 /* @flow */
+import { validAddonTypes } from 'core/constants';
+
+// You have to just laugh at this line!
+export type AddonTypeType = validAddonTypes;
```

Comment:
This doesn't look like it needs to be exported anymore. Your patch will be less funny but hey, that's life.